blog/wyniki-badania-plac-w-komunikacji-technicznej-2023/index.mdx

```diff
@@ -0,0 +1,228 @@
+
+## Przygotowanie do zawodu
+
+Większość ankietowanych nauczyło się zawodu w pracy (87%).
+
+<SurveyTable {...require('./data').trainingTableProps} />
+
+12 osób wybrało opcję "Inny kurs lub certyfikat", oto ich odpowiedzi:
+
+<SurveyBulletedList
+  entries={require('./data').trainingBarChartDataset.filter(
+    (item) => item.count === 1
+  )}
+/>
+
+### Kurs ITCQF
+
+Co ciekawe, na trzecim miejscu pojawił się kurs ITCQF - aż 28 osób się nim
+pochwaliło. To duży skok w porównaniu z poprzednim rokiem, kiedy ten kurs
+wspomniało tylko 9 osób.
+
+<SurveyTable {...require('./data').itcqfTableProps} />
+
+Bardzo nas cieszy, że coraz więcej osób decyduje się na uczestnictwo w tym
+kursie, ale mamy też świadomość, że ten udział procentowy we wszystkich
+odpowiedziach nie jest odzwierciedleniem rzeczywistości. Osoby, które przeszły
+kurs zapewne mają w swoich kręgach na mediach społecznościowych osoby powiązane
+z ITCQF i usłyszały o naszym portalu podczas szkolenia. Dzięki temu dotarli do
+informacji o ankiecie i prawdopodobnie dlatego tak sporo osób z kursem ITCQF
+wzięło w niej udział. Gdybyśmy zebrali więcej odpowiedzi, prawdopodobnie udział
+procentowy osób z kursem ITCQF byłby dużo niższy.
+
+### Doświadczenie
+
+Jak widać poniżej, doświadczenie ma dość znaczący i pozytywny wpływ na to jakie
+otrzymujemy wynagrodzenie.
+
+<SurveyTable {...require('./data').experienceAndEarningsTableProps} />
+
+### Wykształcenie
+
+Poniższa tabelka odpowiada na pytanie czy rodzaj wykształcenia wpływa na
+zarobki.
+
+<SurveyTable {...require('./data').educationEarningsTableProps} />
+
+Wygląda na to, że:
+
+- Przeważają humaniści
+- Nie ma znaczącej różnicy między zarobkami osób po studiach technicznych a osób
+  po studiach humanistycznych
+- Osoby, które mają jednocześnie wykształcenie techniczne i humanistyczne nie
+  zarabiają więcej ale jednocześnie jest to bardzo mała próbka
+
+## Firma
+
+<SurveyPieChart {...require('./data').companyTypePieChartProps} />
+
+Ogromna większość uczestników ankiety pracuje w firmie międzynarodowej, bo aż
+86%. Ten podział ma odzwierciedlenie w wysokości zarobków i są one trochę wyższe
+w firmach zagranicznych.
+
+<SurveyTable {...require('./data').companyTableProps} />
+
+<SurveyPieChart {...require('./data').companySizePieChartProps} />
+
+Jak pokazuje powyższy wykres, większość (choć nieznaczna) z nas pracuje w dużych
+firmach, w których jest ponad 1000 pracowników.
+
+Co ciekawe, nikt nie wybrał odpowiedzi "prywatna działalność" i "2-10 osób". Czy
+to oznacza, że takie firmy raczej nie zatrudniają tech writerów, czy bardziej
+to, że nasze zasięgi nie docierają do takiego typu organizacji? 🤔
+
+Jak te dane wpływają na zarobki? Oto wyniki:
+
+<SurveyTable {...require('./data').companySizeTableProps} />
+
+Wygląda na to, że najlepiej zarabia się w firmach od 11 do 100 osób, a więc
+raczej małych.
+
+<SurveyTable {...require('./data').contractTypeTableProps} />
+
```

Comment:
Mam zastrzeżenie do tej tabeli - dlaczego jest rozróżnienie na B2B (30h) i B2B (samozatrudnienie)? Nie powinno być takich opcji wyboru, bo to wypacza wyniki - jest się na B2B (samozatrudnienie) i koniec - raz się pracuje więcej, raz mniej i skąd te 30 h tygodniowo się wzięło? Ta jedna osoba powinna być dodana do grupy B2B, bez żadnego wyłączania, zwłaszcza jeśli nie było wyboru w pytaniu, a ludzie wpisywali to "z ręki". Takie coś nie wnosi niczego istotnego, a zaburza wyliczenia zarobków. Jak dla mnie powinna być przerobiona tabelka i przeliczony wynik dla całego B2B.